multi-agent/tests/unit/resources/conftest.py

```
@@ -0,0 +1,305 @@
+        return 0
+
+    def group_count(
+        self, identity: Identity, group_by: List[str], filter: Optional[dict] = None,
+    ) -> list:
+        return []
+
+    def delete_by_identity(self, identity: Identity) -> int:
+        return 0
+
+    def find_all_builtins(
+        self, category: Optional[str] = None, resource_type: Optional[str] = None,
```

Ruff (0.16.0)

[warning] Unused method argument: `category`

(ARG002)